### PR TITLE
test: add test for create eth rollup

### PR DESCRIPTION
**Pull request by spsjvc**

### CODE REVIEW

File: src/createRollup.integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { privateKeyToAccount } from 'viem/accounts';
+
+import { nitroTestnodeL2 } from './chains';
+import { generateChainId } from './utils';
+import { prepareChainConfig } from './prepareChainConfig';
+import { createRollupPrepareConfig } from './createRollupPrepareConfig';
+import { createRollupPrepareTransactionRequest } from './createRollupPrepareTransactionRequest';
+import { createRollupPrepareTransactionReceipt } from './createRollupPrepareTransactionReceipt';
+
+const deployerPrivateKey = '0xb6b15c8cb491557369f3c7d2c287b053eb229daa9c22138887752191c9520659';
```
Comment (fionnachan) [inline src/createRollup.integration.test.ts:12]: i think this should be in .env

File: src/createRollup.integration.test.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+const deployer = privateKeyToAccount(deployerPrivateKey);
+
+const client = createPublicClient({
+  chain: nitroTestnodeL2,
+  transport: http(),
+});
+
+it('succesfully deploys eth rollup', async () => {
```
Comment (fionnachan) [inline src/createRollup.integration.test.ts:20]:
```suggestion
it('successfully deploys eth rollup', async () => {
```

File: src/createRollup.integration.test.ts [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
+const batchPoster = deployer.address;
+const validators = [deployer.address];
+
 const publicClient = createPublicClient({
   chain: nitroTestnodeL2,
   transport: http(),
 });
 
-it('successfully deploys eth rollup', async () => {
+it(`successfully deploys core contracts through rollup creator`, async () => {
```
Comment (fionnachan) [inline src/createRollup.integration.test.ts:24]: why are they backticks now?

**Review by fionnachan — APPROVED** [review]

File: src/createRollup.integration.test.ts (hunk 3 above)
Comment (spsjvc) [reply src/createRollup.integration.test.ts:24]: I like backticks, means I can add quotes freely

Comment (fionnachan) [reply src/createRollup.integration.test.ts:24]: lol ok

**Pull request merged**